### PR TITLE
bump iviewer 0.14.0

### PR DESCRIPTION
**Pull request by will-moore**
bump iviewer 0.14.0

cc @jburel

### CODE REVIEW

**Comment by will-moore** [comment]
Build is failing `Molecule` test:

```
  File "/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/urllib3/poolmanager.py", line 326, in connection_from_context
    raise URLSchemeUnknown(scheme)
urllib3.exceptions.URLSchemeUnknown: Not supported URL scheme http+docker
```

@jburel Any ideas what's going on here?

**Comment by sbesson** [comment]
Pretty sure the build issues are related to the issues discussed in https://github.com/ansible-community/molecule-plugins/pull/257 , https://github.com/docker/docker-py/issues/3256 and https://github.com/psf/requests/issues/6707 for more background

As a workaround, @khaledk2 capped the version of `requests` installed in the workflow - see https://github.com/IDR/deployment/pull/424#issuecomment-2146081892.

**Comment by will-moore** [comment]
Rebased and force-pushed. Build now green

**Review by sbesson — APPROVED** [review]

**Review by will-moore — COMMENTED** [review]
The iviewer upgrade is working as expected on idr-next now.
(I can't approve my own PR but I can give it a 👍 )

File: ansible/group_vars/omero-hosts.yml [hunk 1]
```diff
@@ -237,7 +237,7 @@ omero_web_config_set:
 
 omero_web_apps_packages:
 - omero-mapr==0.5.2
-- omero-iviewer==0.13.0
+- omero-iviewer==0.14.0
 - idr-gallery==3.11.0
 - omero-figure==6.2.0
```
Comment (jburel) [inline ansible/group_vars/omero-hosts.yml:242]: Figure is now at 7.0.0 but not really used in IDR

**Pull request merged**